package.json

```diff
@@ -80,7 +83,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run prepush && npm run security-checks",
+      "pre-commit": "npm run build && npm run security-checks",
```

Comment:
😕  Why change this? running a build before we commit?

Reply:
on line 87 (below), we were already running prepush hook that runs test and tslint before we push the code.
The build takes a few seconds whereas the unit tests a few mins (on prepush hook).
It gives some extra assurance that the code is compiled correctly.  Often with Ts, the compiler fails to perform the static analysis of the code (because after all typescript is some syntactic sugar for javascript/es) and  do not flag the errors when parsing the new changes. We run build to make sure we start from a clean slate

Reply:
👍